.buildkite/pipeline.yml

```diff
@@ -9,8 +9,11 @@ steps:
   - command: nix-build ci.nix -A xrefcheck-lib-and-tests
     label: Library and tests
 
-  - command: nix run -f ci.nix pkgs.bats pkgs.diffutils xrefcheck-static -c bash -c "cd tests/golden && bats ./"
-    label: Golden tests (bats)
+  - command: nix run -f ci.nix pkgs.bats pkgs.diffutils xrefcheck-static -c bash -c "cd tests/golden/check-localhost && bats ./"
+    label: Golden tests for localhost (bats)
+
+  - command: nix run -f ci.nix pkgs.bats pkgs.diffutils xrefcheck-static -c bash -c "cd tests/golden/check-cli && bats ./"
+    label: Golden tests for CLI options (bats)
```

Comment:
Oh, we should just have one CI step for running all bats tests.

Reply:
You can do `cd tests/golden && bats ./**` to run all bats tests in all subdirs.

On the other hand, this does mean that the "working directory" for all tests will be `tests/golden`, so they'll probably start failing. You can do what we did in coffer:

* Create a helper file with a setup function that changes the working dir ([example](https://github.com/serokell/coffer/blob/main/tests/golden/helpers.bash))
* Add `load '../helpers` at the top of every bats file ([example](https://github.com/serokell/coffer/blob/main/tests/golden/delete-field-command/delete-field-command.bats))